packages/web-components/src/components/card/card-footer.ts

```diff
@@ -38,13 +43,27 @@ class DDSCardFooter extends DDSLinkWithIcon {
     return Boolean(parentHref) && (!href || parentHref === href);
   }
 
+  /**
+   * Handles `slotchange` event on the default `<slot>`.
+   */
+  protected _handleSlotChange({ target }: Event) {
+    const hasContent = (target as HTMLSlotElement)
+      .assignedNodes()
+      .some(node => node.nodeType !== Node.TEXT_NODE || node!.textContent!.trim());
+    this._hasCopy = hasContent;
+    this.requestUpdate();
```

Comment:
In case a derived class uses this `._handleSlotChange()` for other `<slot>`s in future:

```suggestion
    if (!(target as HTMLSlotElement).name) {
      const hasContent = (target as HTMLSlotElement)
        .assignedNodes()
        .some(node => node.nodeType !== Node.TEXT_NODE || node!.textContent!.trim());
      this._hasCopy = hasContent;
      this.requestUpdate();
    }
```